Representable/representable
# Update selection tool UX

as per discussion in team sync + #423 
**changes**
- removed number for radius of selection tool
- changed to "selection tool size" from "selection size"
- slider thumb changes in size based upon overall selection size

**to test**
- go to entry page
- try out the selection tool, confirm no errors

**screenshots**
small
![image](https://user-images.githubusercontent.com/41943646/95889808-ec6c5700-0d50-11eb-904c-8c9e467b6842.png)
bigger
![image](https://user-images.githubusercontent.com/41943646/95889831-f7bf8280-0d50-11eb-9e86-a5cf1d0adc84.png)


note: it's ever-so-slightly off center vertically... this is because css is really annoying about aligning this item. i think it's probably ok for now! but, we should keep an eye on..
